cookbook/core/containerization/raw_container.py

```diff
@@ -4,71 +4,172 @@
-# output variables
-square = ContainerTask(
-    name="square",
+# output variables.
+calculate_ellipse_area_shell = ContainerTask(
+    name="ellipse-area-metadata-shell",
     input_data_dir="/var/inputs",
     output_data_dir="/var/outputs",
-    inputs=kwtypes(val=int),
-    outputs=kwtypes(out=int),
-    image="alpine",
+    inputs=kwtypes(a=float, b=float),
+    outputs=kwtypes(area=float, metadata=str),
+    image="rawcontainers-shell:v1",
     command=[
-        "sh",
-        "-c",
-        "echo $(( {{.Inputs.val}} * {{.Inputs.val}} )) | tee /var/outputs/out",
+        './calculate-ellipse-area.sh',
+        '/var/inputs',
+        '/var/outputs',
     ],
 )
 
+calculate_ellipse_area_python = ContainerTask(
+    name="ellipse-area-metadata-python",
+    input_data_dir="/var/inputs",
+    output_data_dir="/var/outputs",
+    inputs=kwtypes(a=float, b=float),
+    outputs=kwtypes(area=float, metadata=str),
+    image="rawcontainers-python:v1",
+    command=[
+        'python',
+        'calculate-ellipse-area.py',
+        '/var/inputs',
+        '/var/outputs',
+    ],
+)
 
-sum = ContainerTask(
-    name="sum",
-    input_data_dir="/var/flyte/inputs",
-    output_data_dir="/var/flyte/outputs",
-    inputs=kwtypes(x=int, y=int),
-    outputs=kwtypes(out=int),
-    image="alpine",
+calculate_ellipse_area_r = ContainerTask(
+    name="ellipse-area-metadata-r",
+    input_data_dir="/var/inputs",
+    output_data_dir="/var/outputs",
+    inputs=kwtypes(a=float, b=float),
+    outputs=kwtypes(area=float, metadata=str),
+    image="rawcontainers-r:v1",
     command=[
-        "sh",
-        "-c",
-        "echo $(( {{.Inputs.x}} + {{.Inputs.y}} )) | tee /var/flyte/outputs/out",
+        'Rscript',
+        '--vanilla',
+        'calculate-ellipse-area.R',
+        '/var/inputs',
+        '/var/outputs',
     ],
 )
 
+calculate_ellipse_area_haskell = ContainerTask(
+    name="ellipse-area-metadata-haskell",
+    input_data_dir="/var/inputs",
+    output_data_dir="/var/outputs",
+    inputs=kwtypes(a=float, b=float),
+    outputs=kwtypes(area=float, metadata=str),
+    image="rawcontainers-haskell:v1",
+    command=[
+        './calculate-ellipse-area',
+        '/var/inputs',
+        '/var/outputs',
+    ],
+)
+
+calculate_ellipse_area_julia = ContainerTask(
+    name="ellipse-area-metadata-julia",
+    input_data_dir="/var/inputs",
+    output_data_dir="/var/outputs",
+    inputs=kwtypes(a=float, b=float),
+    outputs=kwtypes(area=float, metadata=str),
+    image="rawcontainers-julia:v1",
+    command=[
+        'julia',
+        'calculate-ellipse-area.jl',
+        '/var/inputs',
+        '/var/outputs',
+    ],
+)
+
+@task
+def report_all_calculated_areas(
+        area_shell: float, metadata_shell: str,
+        area_python: float, metadata_python: str,
+        area_r: float, metadata_r: str,
+        area_haskell: float, metadata_haskell: str,
+        area_julia: float, metadata_julia: str,
```

Comment:
nit: `black` would format it this way:

```suggestion
    area_shell: float, metadata_shell: str,
    area_python: float, metadata_python: str,
    area_r: float, metadata_r: str,
    area_haskell: float, metadata_haskell: str,
    area_julia: float, metadata_julia: str,
```

Reply:
(we should probably to some pre-commit hook on these files at some point)

Reply:
I will work on this in a separate PR.